src/main/java/org/isf/medicalstockward/manager/MovWardBrowserManager.java

```diff
@@ -295,4 +295,51 @@ public int compare(MovementForPrint o1, MovementForPrint o2) {
 		}
 	}
 
+	/* * 
+	 * Delete the MovementWard for the specified criteria.
+	 *
+	 * @param movWard - the movement ward
+	 * @throws OHServiceException if an error occurs retrieving the medical.
+	 */
+	public void deleteLastMovementWard(MovementWard movWard) throws OHServiceException {
+		MovementWard movementWard = ioOperations.getLastMovementWard(movWard.getWard());
+		System.out.println();
+		if (movementWard.getCode() != movWard.getCode()) {
+			System.out.println("Erreur code incorrect");
+			throw new OHDataValidationException(
+							new OHExceptionMessage(MessageBundle.getMessage("angal.medicalstock.onlythelastmovementcanbedeleted.msg")));
+		}
+		if (movementWard.getWardTo() != null) {
```

Comment:
Shouldn't this be like this? 

```suggestion
		if (movWard.getWardTo() != null) {
```

Maybe it is more readable if you rename `movWard` -> `movWardToDelete` and `movementWard` -> `lastMovementWard`

Reply:
okay